pkg/name/name.go

```diff
@@ -228,16 +228,21 @@ func GetFromTreePath(inputTree map[string]interface{}, path util.Path) (interfac
 		return val, true, nil
 	}
 	switch newRoot := val.(type) {
+	case string:
+		return nil, false, nil
 	case map[string]interface{}:
 		return GetFromTreePath(newRoot, path[1:])
 	case []interface{}:
 		for _, node := range newRoot {
-			nextVal, found, err := GetFromTreePath(node.(map[string]interface{}), path[1:])
-			if err != nil {
-				continue
-			}
-			if found {
-				return nextVal, true, nil
+			switch nt := node.(type) {
+			case map[string]interface{}:
```

Comment:
This is probably ok for now, but the right fix is for this function to take an interface{} - then it should handle a slice, map or scalar separately. In fact I think there's already such a version floating around... 
But since there's not much time, feel free to leave a TODO or create an issue.

Reply:
Fixed